environment.yml

```diff
@@ -23,7 +23,7 @@ dependencies:
     - black
     - blackdoc
     - docformatter
-    - flake8
+    - flakeheaven
```

Comment:
```suggestion
    - flakeheaven>=2
```

Reply:
Updated the pin to `flakeheaven>3` in 7b32a52ba6595f03cb1f632e201632db65ebb159 to include a fix for the `max_line_length` setting. See https://github.com/flakeheaven/flakeheaven/releases/tag/3.0.0